### PR TITLE
Remove warning log messages for Long Running Operations

### PR DESCRIPTION
**Pull request by derekbekoe**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@derekbekoe__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

File: src/azure/cli/commands/__init__.py [hunk 1]
```diff
@@ -89,7 +89,6 @@ def _delay(self):
         time.sleep(self.poll_interval_ms / 1000.0)
 
     def __call__(self, poller):
-        logger.warning(self.start_msg)
```
Comment (yugangw-msft) [inline src/azure/cli/commands/__init__.py]: i like this change personally, but for the people get used to interactive background, is it possible we can still output the starting message, but just don't prefix with "warning:"?

**Comment by tjprescott** [comment]
:shipit:

File: src/azure/cli/commands/__init__.py (hunk 1 above)
Comment (derekbekoe) [inline src/azure/cli/commands/__init__.py]: 'WARNING:' is not displayed in interactive mode (if there's a TTY available).
But log message are prefixed with the log level if there is no TTY (e.g. redirecting the output to file) as was the case in #572.

**Pull request merged**